script.js

```diff
@@ -1,4 +1,74 @@
+//Start with a default mode
+var currentMode = "Roll Dice";
+
+//Set up an array for the dice rolls outcome
+var diceArray = [];
+
+//Array for player 1 at position 0, and player 2 at position 1
+var playersFinalNumber = [];
+
+//Random Dice roll
+var diceRoll = function () {
+  return Math.floor(Math.random() * 6) + 1;
+};
+
+//Mode 1, two dice rolls, added to array [roll1,roll2]
+var modeOneDiceRolls = function () {
+  diceArray = [diceRoll(), diceRoll()];
+  return diceArray;
+};
+
+//Mode 2, ordering dice, return as string to combine them
+var diceOrderd = function (firstDice) {
```

Comment:
Try to name your functions using verbs like orderDice because functions are essentially 'actions'.